### PR TITLE
Do not take ownership of the attributes

### PR DESCRIPTION
**Pull request by A6GibKm**
This will free us of cloning them.

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Would need a rebase

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by A6GibKm** [comment]
Rebased and tested with the CLI utility.

File: cli/src/main.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ use oo7::{
     zbus,
 };
 
-const BINARY_NAME: &'static str = env!("CARGO_BIN_NAME");
+const BINARY_NAME: &str = env!("CARGO_BIN_NAME");
```
Comment (bilelmoussaoui) [inline cli/src/main.rs:14]: recent rust complains about this

Comment (A6GibKm) [reply cli/src/main.rs]: Removed, if needed can be done elsewhere.

**Pull request merged**